docs/integrations/spark.md

```diff
@@ -55,42 +51,31 @@ interact with your data on lakeFS.
 
   </div>
   <div markdown="2" id="install-databricks">
+In  your cluster settings, under the _Libraries_ tab, add the following Maven package:
 
-**Note** For your convenience, follow this [step by step Databricks integration tutorial, including lakeFS Hadoop File System, Python client and lakeFS SPARK client](https://lakefs.io/blog/databricks-lakefs-integration-tutorial/).
```

Comment:
We don't have the tutorial any longer?  If we do, we may want to find some of of linking to it.

Reply:
It appears later, in the configuration section